client/conf/log4j-cloud.xml.in

```diff
@@ -50,9 +50,7 @@ under the License.
       <!-- Append warnings+ to the syslog if it is listening on UDP port -->
       <!-- ============================== -->
 
-      <Syslog name="SYSLOG" host="localhost" facility="LOCAL6">
-         <ThresholdFilter level="WARN" onMatch="ACCEPT" onMismatch="DENY"/>
-         <PatternLayout pattern="%d{DEFAULT} %-5p [%c{1.}] (%t:%x) (logid:%X{logcontextid}) %m%ex{filters(${filters})}%n"/>
+      <Syslog name="SYSLOG" host="localhost" port="514" protocol="UDP" appName="ApacheCloudStack" facility="LOCAL6">
       </Syslog>
```

Comment:
```suggestion
      <Syslog name="SYSLOG" host="localhost" port="514" protocol="UDP" appName="ApacheCloudStack" facility="LOCAL6">
               <ThresholdFilter level="WARN" onMatch="ACCEPT" onMismatch="DENY"/>
               <PatternLayout pattern="%d{DEFAULT} %-5p [%c{1.}] (%t:%x) (logid:%X{logcontextid}) %m%ex{filters(${filters})}%n"/>
      </Syslog>
```
This works fine on my env, assuming that rsyslog is configured to listen on port 514

Reply:
ok, I’m fine with keeping them. I just wanted the simplest configuration that would work, and leave it to the users to extend.

Reply:
@DaanHoogland I understand your point, I'm just worried that changing the default pattern might disrupt users that are parsing these logs and expect this patern.